### ISSUE TITLE
the class_loss loc_loss mbox_loss are 'NaN'

### ISSUE THREAD

**Issue by daibu0** [albanie/mcnSSD#5]
hello, when I trained the ssd by your code, in the iteration 1, the class_loss loc_loss mbox_loss are 21.07 2.914 23.989, but after the iteration 1 , all of the loss turned to 'NaN', Do you know how to solve it? thank you!

**Comment by albanie** [comment]
Hi @daibu0, it sounds as though the learning rate is too high.  Normally the first epoch uses a lower "warmup" learning rate (it is set [here](https://github.com/albanie/mcnSSD/blob/master/pascal/ssd_pascal_train.m#L58-L77)).  You could try reducing the learning rate that follows.

**Comment by daibu0** [comment]
@albanie thanks for your reply, i have solved this problem. i use your default code to train the ssd and i use cpu. so the derOutputs of the mbox_loss is inf, and now i have modified it.

**Comment by albanie** [comment]
Great, glad to hear it!
